src/app/reviewer/[memberSlug]/study/[studyIdentifier]/review/study-review-buttons.tsx

```diff
@@ -41,18 +43,22 @@ export const StudyReviewButtons: FC<{ study: SelectedStudy; memberSlug: string }
 
     if (study.status === 'APPROVED' && study.approvedAt) {
         return (
-            <Group c="#12B886" gap="0">
-                <CheckCircle weight="fill" />
-                <Text>Approved on {dayjs(study.approvedAt).format('MMM DD, YYYY')}</Text>
+            <Group gap="0.5rem">
+                <CheckCircle weight="fill" size={24} color={theme.colors.green[9]} />
+                <Text fz="xs" fw={600} c="green.9">
+                    Approved on {dayjs(study.approvedAt).format('MMM DD, YYYY')}
+                </Text>
             </Group>
         )
     }
 
     if (study.status === 'REJECTED' && study.rejectedAt) {
         return (
-            <Group c="#FA5252" gap="0">
-                <XCircle weight="fill" />
-                <Text>Rejected on {dayjs(study.rejectedAt).format('MMM DD, YYYY')}</Text>
+            <Group gap="0.5rem">
```

Comment:
it's _slightly_ more efficient to use one of the [built-in gap sizes](https://getbootstrap.com/docs/4.4/utilities/spacing/#notation) because they'll use a pre-defined css class, and a raw value like this will cause a style to be defined directly on the tag:
```suggestion
            <Group gap="sm">
```